PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+Fixes # (issue number, if applicable) 
+
+### Description
+> Please include a summary of the change, relevant motivation and context. If possible and applicable, include before and after screenshots and a URL where the changes can be seen.
+    
+### Caveats
+
+> Please list anything which has been left out of this PR or which should be considered before this PR is accepted
+Check any of the following which apply:
+
+* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
+* [ ]  This change requires a documentation update
+    * [ ]  I have made corresponding changes to the documentation
+* [ ]  This change requires new dependencies (please list)
+
+if you make changes to the python code
+  
+* [ ]  my code passes [tox](https://tox.readthedocs.io/en/latest/) check, you can receive warnings about tests, documentation or both
```

Comment:
Some confusion about the intended meaning of "you" here.

Reply:
I have changed it to "I"